packages/@react-aria/interactions/test/usePress.test.js

```diff
@@ -21,9 +21,9 @@ import {theme} from '@react-spectrum/theme-default';
 import {usePress} from '../';
 
 function Example(props) {
-  let {elementType: ElementType = 'div', ...otherProps} = props;
+  let {elementType: ElementType = 'div', style, ...otherProps} = props;
   let {pressProps} = usePress(otherProps);
-  return <ElementType {...pressProps} tabIndex="0">test</ElementType>;
+  return <ElementType style={style} {...pressProps} tabIndex="0">test</ElementType>;
```

Comment:
Note: consumers of usePress will need to make sure that they apply their style obj before pressProps otherwise the `user-select:none` from usePress will be overwritten

Reply:
Maybe https://github.com/adobe/react-spectrum/issues/2507?

Reply:
Yep, that change to mergeProps should help when implemented